workitem/workitemtype_repository.go

```diff
@@ -181,6 +182,21 @@ func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UU
 	return &app.WorkItemTypeSingle{Data: &result}, nil
 }
 
+// List returns work item types that derives from PlannerItem type
+func (r *GormWorkItemTypeRepository) ListPlannerItems(ctx context.Context, spaceID uuid.UUID) ([]WorkItemType, error) {
+	var rows []WorkItemType
+	db := r.db.Select("id").Where("space_id = ? AND path::text LIKE '"+LtreeSafeID(SystemPlannerItem)+".%'", spaceID.String())
```

Comment:
@pranavgore09 Sure! I'll think about having a function in path because it is weird. I need to create a path object to get a String. Maybe a path.NewPath(uuid) returns path ...